src/aap_eda/settings/default.py

```diff
@@ -517,7 +517,9 @@ def get_rq_queues() -> dict:
 # Otherwise, the default name is used
 RULEBOOK_QUEUE_NAME = settings.get("RULEBOOK_QUEUE_NAME", "activation")
 
-RQ_STARTUP_JOBS = []
+RQ_STARTUP_JOBS = [
+    {"func": "aap_eda.tasks.analytics.schedule_gather_analytics"},
```

Comment:
we should define here "job_id" argument. It will prevent potential duplicates or race conditions with restarts of the scheduler same as we do with other scheduled jobs. 
(notice that here is not "id" param, but "job_id" because it uses `scheduler.enqueue_at`